question/bank/managecategories/templates/basecategory.mustache

```diff
@@ -0,0 +1,54 @@
+{{!
+    This file is part of Moodle - http://moodle.org/
+
+    Moodle is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Moodle is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+}}
+{{!
+    @template qbank_managecategories/basecategory
+
+    This template renders the base for category view.
+
+    Example context (json):
+    {
+        "helpstringhead": "<h2> Edit categories <a href="someurl">...</a></h2>",
+        "checkbox": "<form id="qbshowdescr-form"...> ... </form>",
+        "categoriesrendered": "<div id="categoriesrendered"> ... </div>",
+    }
+}}
+
+<div class="row">
+    <div class="pl-4">
+        {{{helpstringhead}}}
+    </div>
+</div>
+<br>
+<div class="row">
+    <div class="pl-4">
+        <button type="submit" data-action="addcategory" class="btn btn-primary"> 
```

Comment:
need capability check around this button, for both webservice and mustache

Reply:
fixed